enh(oidc): improve groups & roles management (#2408)
centreon/centreon-documentation: versioned_docs/version-23.10/connect/openid.md

}
```

> When retrieving attributes, Centreon will merge the attributes retrieved in the JWT (JSON Web Tokens) with those

> > retrieved in the selected entry point, giving priority to the JWT attributes. Thus, if an attribute is available in

Reviewer: ```suggestion
> retrieved from the selected entry point, giving priority to the JWT attributes. Thus, if an attribute is available in
```

> > the JWT and in the selected entry point, then the attribute selected will be the one coming from the JWT.

Reviewer: ```suggestion
> the JWT and in the selected entry point, then the selected attribute will be the one coming from the JWT.
```

}
```

> > When retrieving attributes, Centreon will merge the attributes retrieved in the JWT (JSON Web Tokens) with those

Reviewer: ```suggestion
> When retrieving attributes, Centreon will merge the attributes retrieved from the JWT (JSON Web Token) with those
```

> > retrieved in the selected entry point, giving priority to the JWT attributes. Thus, if an attribute is available in

Reviewer: ```suggestion
> retrieved from the selected entry point, giving priority to the JWT attributes. Thus, if an attribute is available in
```
